contracts/lib/SignatureVerification.sol

```diff
@@ -72,10 +75,15 @@ contract SignatureVerification is SignatureVerificationErrors, LowLevelHelpers {
 
                 // Place final byte on the stack at v.
                 v := byte(0, mload(add(signature, ThreeWords)))
+
+                // Whether v is 27 or 28.
+                // The magic number has the 27th and 28th bytes
+                // counting from the most significant byte set to 1.
+                vIsValid := byte(v, 0x0101000000)
```

Comment:
So this supposedly works by seeing if byte 27 or 28 is set in the value, but the confusing part is that `byte` works "left aligned" while the literal is "right aligned".

Should have the full 256 bits as a constant for readability, and I'd argue against moving this into a constants file as that reduces readability.

Reply:
I do think that it's an improvement to readability to have the named constant, as it answers the question of "what is this value" in a succinct way (yes, if you're already familiar with the EVM at a deep level and "know" what the value is then it's useful to have it inlined but most readers will not have that deep familiarity). It's also consistent with the rest of the codebase.

Reply:
I mean `TwentySeventhAndTwentyEighthBytesSet` is pretty on the nose and can be evaluated for accuracy in an isolated context...

Reply:
(or something to that effect)

Reply:
I hear you that now a reviewer has to go and look up that constant to ensure that it's correct, but the same can be said for masks and offsets